api/src/pcapi/routes/backoffice_v3/templates/titelive/search_result.html

```diff
@@ -56,17 +56,25 @@ <h5 class="card-title">{{ json.oeuvre.titre }}</h5>
                         <b>Taux TVA :</b> {{ article.taux_tva }} %
                       </li>
                       <li class="list-group-item">
-                        <b>Code CLIL :</b> {{ article.code_clil }}
+                        <b>Code CLIL :</b>
+                        {% if article.code_clil %}
+                          {{ article.code_clil }}
+                        {% else %}
+                          Non renseigné
+                        {% endif %}
```

Comment:
Je ne suis pas expert en Jinja, mais ne pourrait-on pas utiliser `default`, ici ? Ça semble plus concis.

```suggestion
                        <b>Code CLIL :</b> {{ article.code_clil | default('Non renseigné')}}
```